pylint/testutils/output_line.py

```diff
@@ -93,8 +93,8 @@ def from_msg(cls, msg: Message, check_endline: bool = True) -> "OutputLine":
 
     @staticmethod
     def _get_column(column: str) -> int:
-        """Handle column numbers with the exception of pylint < 3.8 not having them
-        in the ast parser.
+        """Handle column numbers except for python < 3.8. The ast parser in those versions does
```

Comment:
```suggestion
        """Handle column numbers except for python < 3.8. The ast parser in those versions doesn't
```

I suggested poorly 😅